### PR TITLE
PMM-10626: pmm server install command

### PR DESCRIPTION
**Pull request by michal-kralik**
PMM-10626

Build: SUBMODULES-2785

### CODE REVIEW

File: build/scripts/build-client-binary [hunk 1]
```diff
@@ -128,6 +128,11 @@ main() {
 
     rm -rf ${binary_tarball}
     mkdir -p $(dirname ${binary_tarball}) || :
+    
+    if [[ $keep_pmm_binary != "1" ]]; then
```
Comment (atymchuk) [inline build/scripts/build-client-binary:132]: If `pmm_version` is global, then we could put the code here from https://github.com/percona/pmm/pull/1207/files#diff-f6033d471a2409e0d14b46e8bb0d7ad855b5093ff9c109246c388706e819ed76R53-R55

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Review by rishat-ishbulatov — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]
LGTM

File: admin/commands/pmm/server/docker/install.go [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+	return `
+	
+PMM Server is now available at http://localhost/
+
+User: admin
+Password: ` + r.adminPassword
+}
+
+// ErrDockerNoAccess is returned when there is no access to Docker or Docker is not running.
+var ErrDockerNoAccess = fmt.Errorf("DockerNoAccess")
```
[the remark this answers is not in the document]
Comment (michal-kralik) [reply admin/commands/pmm/server/docker/install.go:70]: Originally I wanted to have a more descriptive error message but then realized most of the time we have the error message in format `%w: %s`.
This turns the error message into something like:
`DockerNoAccess: you have no access to Docker. Try running as root.`
versus
`no access to Docker: you have no access to Docker. Try running as root.`

This is consistent with other parts of the code.
Maybe a better error would be 
`You have no access to Docker. Try running as root (ERR_DOCKER_NO_ACCESS)`

or something along these lines.
Looking for feedback here.

[the remark this answers is not in the document]
Comment (atymchuk) [reply admin/commands/pmm/server/docker/install.go:70]: I quite like the version of `DockerNoAccess: you have no access to Docker. Try running as root.`

**Review by atymchuk — APPROVED** [review]

**Pull request merged**